manage.py

```diff
@@ -129,5 +135,47 @@ def setup_general():
     Agency.insert_agencies()
     EditableHTML.add_default_faq()
 
+
+@manager.command
+def run_worker():
+    """Initializes a slim rq task queue."""
+    listen = ['default']
+    conn = Redis(
+        host=app.config['RQ_DEFAULT_HOST'],
+        port=app.config['RQ_DEFAULT_PORT'],
+        db=0,
+        password=app.config['RQ_DEFAULT_PASSWORD']
+    )
+
+    with Connection(conn):
+        worker = Worker(map(Queue, listen))
+        worker.work()
+
+
+@manager.command
+def run_scheduler():
+    """Initializes a rq scheduler."""
+    conn = Redis(
+        host=app.config['RQ_DEFAULT_HOST'],
+        port=app.config['RQ_DEFAULT_PORT'],
+        db=0,
+        password=app.config['RQ_DEFAULT_PASSWORD']
+    )
+
+    setup_loghandlers('INFO')
+    scheduler = Scheduler(connection=conn, interval=60.0)
+    for _ in xrange(10):
+        try:
+            scheduler.run()
+        except ValueError, exc:
```

Comment:
Nit: can you use `as` syntax instead of the comma here? Actually hadn't seen the comma before. http://stackoverflow.com/questions/2535760/python-try-except-comma-vs-as-in-except